igor-web/src/main/groovy/com/netflix/spinnaker/igor/wercker/WerckerBuildMonitor.groovy

```diff
@@ -187,11 +183,12 @@ class WerckerBuildMonitor extends CommonPollingMonitor<PipelineDelta, PipelinePo
                     runningBuilds: currentlyBuilding
                     ))
         } catch (e) {
-            log.error("Error processing runs for [{}:{}]", kv("master", master), kv("pipeline", pipeline), e)
-            if (e.cause instanceof RetrofitError) {
-                def re = (RetrofitError) e.cause
-                log.error("Error communicating with Wercker for [{}:{}]: {}", kv("master", master), kv("pipeline", pipeline), kv("url", re.url), re)
-            }
+          log.error("Error processing runs for [{}:{}]", kv("master", master), kv("pipeline", pipeline), e)
+          if (e instanceof SpinnakerHttpException) {
+            log.error("Error communicating with Wercker for [{}:{}]: {}", kv("master", master), kv("pipeline", pipeline), kv("url", e.url), e)
+          }else if (e instanceof SpinnakerServerException) {
+            log.error("Error communicating with Wercker for [{}:{}]: {}", kv("master", master), kv("pipeline", pipeline), e.getCause())
```

Comment:
I don't think `e.getCause()` is interesting here, or at least I think the responsibility is on SpinnakerServerException (or its children) to put the relevant info in the exception's message, so that logging "plain old" `e` is the way forward.  And...there's already a log message above that does that, so I'd say we can ditch this whole if block.